server/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Stage 1: Builder image
+FROM node:8-slim AS builder
```

Comment:
Je pense qu'on peut gagner beaucoup à utiliser alpine. D'autre part on va utiliser node 10.

Reply:
Ok pour node 10 mais pas apk
car en interne seul les cache des dépôts debian et autre distro sont disponible.
La node 10 slim est basée sur debian et diminue déjà en taille.

Reply:
Mais on n'a pas besoin d'installer quoi que ce soit…

Reply:
Hope until we need it 😉
Cf installation de dépendances faisant appel à des librairies ou compilo (  cf crypto ou autre gcc)